Hashlink.js

```diff
@@ -26,24 +26,24 @@ export class Hashlink {
    * @param {Object} options - The options for the create operation.
    * @param {Uint8Array} [options.data] - The data associated with the given URL.
    *   If provided, this data is used to create the cryptographic hash.
-   * @param {Array} options.urls - One or more URLs that contain the data
+   * @param {Array} options.codecs - One or more codecs that should be used
+   *   to encode the data.
+   * @param {Array} [options.urls] - One or more URLs that contain the data
    *   referred to by the hashlink.
-   * @param {Array} options.transforms - One or more URLs that contain the data
-   *   referred to by the hashlink.
-   * @param {Object} options.meta - A set of key-value metadata that will be
+   * @param {Object} [options.meta] - A set of key-value metadata that will be
    *   encoded into the hashlink.
    *
    * @returns {Promise<string>} Resolves to a string that is a hashlink.
    */
-  async create({data, urls, transforms, meta = {}}) {
+  async create({data, urls, codecs, meta = {}}) {
     // ensure data or urls are provided
     if(data === undefined && urls == undefined) {
```

Comment:
Minor fix, but this doesn't even do an "either x or y" test as the comment and error state.  Logic or both texts should be updated.
```suggestion
    if(data === undefined && urls === undefined) {
```